build.sbt

```diff
@@ -27,6 +27,7 @@ scalaVersion := scala212
 crossScalaVersions := supportedScalaVersions
 
 libraryDependencies ++= Seq(
+  "io.delta" %% "delta-core" % "0.6.1",
```

Comment:
I met the following error at `sbt assembly` and no idea how to fix it.
```
[info] Merging files...
[warn] Merging 'META-INF/MANIFEST.MF' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.abego.treelayout/org.abego.treelayout.core/pom.properties' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.abego.treelayout/org.abego.treelayout.core/pom.xml' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/ST4/pom.properties' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/ST4/pom.xml' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/antlr-runtime/pom.properties' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/antlr-runtime/pom.xml' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/antlr4-runtime/pom.properties' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/antlr4-runtime/pom.xml' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/antlr4/pom.properties' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.antlr/antlr4/pom.xml' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.glassfish/javax.json/pom.properties' with strategy 'discard'
[warn] Merging 'META-INF/maven/org.glassfish/javax.json/pom.xml' with strategy 'discard'
scala.MatchError: /Users/eunjin/IdeaProjects/hyperspace_ej/python/run-tests.py (of class java.lang.String)
    at sbtassembly.AssemblyUtils$.sourceOfFileForMerge(AssemblyUtils.scala:23)
    at sbtassembly.MergeStrategy$$anonfun$sbtassembly$MergeStrategy$$filenames$1.apply(MergeStrategy.scala:28)
    at sbtassembly.MergeStrategy$$anonfun$sbtassembly$MergeStrategy$$filenames$1.apply(MergeStrategy.scala:27)
    at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
    at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
    at scala.collection.Iterator$class.foreach(Iterator.scala:727)
    at scala.collection.AbstractIterator.foreach(Iterator.scala:1157)
    at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
    at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
    at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
    at scala.collection.AbstractTraversable.map(Traversable.scala:105)
    at sbtassembly.MergeStrategy$.sbtassembly$MergeStrategy$$filenames(MergeStrategy.scala:27)
    at sbtassembly.MergeStrategy$$anon$6.apply(MergeStrategy.scala:93)
    at sbtassembly.MergeStrategy.apply(MergeStrategy.scala:21)
    at sbtassembly.Assembly$.sbtassembly$Assembly$$applyStrategy$1(Assembly.scala:107)
    at sbtassembly.Assembly$$anonfun$14.apply(Assembly.scala:132)
    at sbtassembly.Assembly$$anonfun$14.apply(Assembly.scala:129)
[error] (*:assembly) scala.MatchError: /<PATH>/hyperspace_ej/python/run-tests.py (of class java.lang.String)
```
and below `assemblyMergeStrategy in assembly` didn't work.

Reply:
Thanks @imback82 for the fix 👍